types/types.go

```diff
@@ -80,8 +80,9 @@ const (
 )
 
 const (
-	DefaultAPIPort           = 9500
-	DefaultWebhookServerPort = 9443
+	DefaultAPIPort               = 9500
+	DefaultConversionWebhookPort = 9443
+	DefaultAdmissionWebhookPort  = 8443
```

Comment:
What's the previous admission webhook port?

Reply:
Besides, we need to update the doc like [this](https://github.com/longhorn/website/pull/681) after the PR merge.

Reply:
both admission and conversion webhook ports were 9443 since they are in different deploy/svc before

but after this PR, both services are in the manager
- conversion webhook: 9443
- admission webhook: 8443

Reply:
Yes, we also need to update the network policy
https://github.com/longhorn/charts/pull/101

Reply:
I am thinking you can use port 9501, 9502, 9503... This is more straightforward when we check the port usage.

Reply:
Sure, sounds good to me!
Fixed.